app/assets/javascripts/directives/validation/error_on_tab.js

```diff
@@ -0,0 +1,22 @@
+ManageIQ.angular.app.directive('errorOnTab', ['$timeout', function($timeout) {
+  return {
+    link: function (scope, elem, attrs) {
+      scope.$on('setErrorOnTab', function(e, args) {
+        if (elem[0].attributes['error-on-tab'].value === args.tab) {
```

Comment:
Yes, the value (tab value) will change here dynamically based on which tab has erroneous fields.